css/index.css

```diff
@@ -37,16 +110,54 @@ body {
   background-color: var(--primary-color);
 }
 
-.header-link {
-  display: none;
-}
-
-.logo {
-  font: 700 22px poppins, sans-serif;
+.nav-container {
+  font: 700 18px poppins, sans-serif;
   color: var(--logo-color);
   display: flex;
   justify-content: space-between;
   align-items: center;
+  margin-top: 8px;
+}
+
+.menu-link {
+  list-style: none;
+  margin-bottom: 8px;
+  font-family: Roboto, sans-serif;
+}
+
+.menu-open-icon {
+  display: flex;
+}
+
+.menu-open-icon:hover {
+  cursor: pointer;
```

Comment:
Why is this element a pointer only when it is hovered? If it's an element that should be pointer then it should with or without hover. I would recommend moving this into general styles. Alternatively, make the element a button and have a general style at the top to make all button elements a pointer cursor and remove all default styles such as background and border.